automation/test.sh

```diff
@@ -179,10 +179,10 @@ echo "Deploying CDI"
 oc apply -f https://github.com/kubevirt/containerized-data-importer/releases/download/$CDI_VERSION/cdi-operator.yaml
 oc apply -f https://github.com/kubevirt/containerized-data-importer/releases/download/$CDI_VERSION/cdi-cr.yaml
 
-oc patch cdi cdi -n cdi --patch '{"spec": {"config": {"dataVolumeTTLSeconds": -1}}}' --type merge
-
 oc wait --for=condition=Available --timeout=${timeout}s CDI/cdi -n cdi
 
+oc patch cdi cdi -n cdi --patch '{"spec": {"config": {"dataVolumeTTLSeconds": -1}}}' --type merge
```

Comment:
What is the reason for this change?

Reply:
@ksimon1 I'm using CodeReady Containers for development on my workstation. While I'm not entirely sure of the underlying reasons, some empirical testing has shown that this change allows the tests to work. Otherwise, they fail when applying the data volumes, as the persistent volume claim is unable to claim any storage.

Additionally, I've encountered the same issue when installing an OpenShift cluster in a VM and running it with QEMU. Do you have any insights into why this might be happening?